### PR TITLE
Catch pokemon with the current available best ball

### PR DESCRIPTION
**Pull request by Cyperghost**

### CODE REVIEW

**Comment by FabianTerhorst** [comment]
Maybe add an parameter to make the masterball excludable, because the masterball is very rare.

File: src/resources/protobuf [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 333e54707ca221324c6b9eedb6a7bc2fcae02270
+Subproject commit 636c52f4df519dd74000bbe8e42b8cb32ba8b51a
```
Comment (FabianTerhorst) [inline src/resources/protobuf]: you updated the protos?

Comment (Cyperghost) [inline src/resources/protobuf]: Yes,there is a problem?

Comment (FabianTerhorst) [inline src/resources/protobuf]: because #419 used a newer version

Comment (Cyperghost) [inline src/resources/protobuf]: Oh, thanks.
I forgot to change the branch ;(

**Comment by FabianTerhorst** [comment]
👍

**Pull request merged**